CHANGELOG.md

```diff
@@ -33,35 +33,38 @@ Release to address the issues found during the GlideinWMS 3.11.1 hackathon.
 
 ### Changed defaults / behaviours
 
--   When using credential generators, glideclient classads are now identified by the generated credential ID.
+-   When using credential generators, glideclient classads are now identified by the generated credential ID. (PR #576)
 
 ### Deprecated / removed options and commands
 
 ### Security Related Fixes
 
 ### Bug Fixes
 
--   Fixed monitoring that had broken due to inconsistent credential IDs used in Glideins.
--   Fixed an issue that would cause the Credentials library to use the wrong timezones when validating X509 certificates.
--   Fixed a bug that would cause CachedGenerator to fail if `cache_dir` wasn't provided in the context.
--   Fixed issues that would prevent a 3.11 Frontend to request Glideins from `batch` entries on a 3.11 Factory.
+-   Fixed monitoring that had broken due to inconsistent credential IDs used in Glideins. (PR #576)
+-   Fixed an issue that would cause the Credentials library to use the wrong timezones when validating X509 certificates. (PR #576)
+-   Fixed a bug that would cause CachedGenerator to fail if `cache_dir` wasn't provided in the context. (PR #576)
+-   Fixed issues that would prevent a 3.11 Frontend to request Glideins from `batch` entries on a 3.11 Factory. (PR #583)
+-   Fixed warning about misspelled local variable name in cvmfs_helper_funcs.sh (PR #597)
 
 ### Testing / Development
 
 ### Known Issues
 
-## v3.10.16 \[2025-09-08\]
+## v3.10.16 \[2025-09-29\]
 
 Bug fix release. Mostly CVMFS and HTCondor job wrapper related.
 
 ### New features / functionalities
 
--   create_cvmfsexec_distros.sh updated adding EL9 to platforms and improved command syntax (PR #582)
+-   create_cvmfsexec_distros.sh updated adding EL9 to platforms and improved command syntax (Issue #549, PR #582)
+-   Added new staticextra resource type – behaves like static (creates one dedicated slot per instance with a virtual CPU each), but instead of subtracting memory from the main partitionable slot, it adds memory to it (Issue #590)
```

Comment:
Info about the PR related to this issue is missing here. I checked the issue referenced and it has the PR linked to it. So, do we safely ignore adding the PR info? I'm thinking about ignoring also because of the fact that 3.10.16 (and its CHANGELOG) has had a final release already